docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+
+FROM alpine:latest AS builder
+
+ARG BUILD_TYPE
+ARG RELEASE
```

Comment:
I'm a bit unfamiliar with docker... I understand most users would want a stable release (e.g. v4.4.2) but what about the latest development (like the git master branch)? I mean is it usually specified in this same dockerfile or in another dockerfile?

Reply:
I will edit the Dockerfile so that it allows to pull both the master and the tagged branches (I'll just put an if)